src/Security/Authentication/Core/src/RemoteAuthenticationOptions.cs

```diff
@@ -137,6 +137,18 @@ public new RemoteAuthenticationEvents Events
     /// Determines the settings used to create the correlation cookie before the
     /// cookie gets added to the response.
     /// </summary>
+    /// <remarks>
+    /// <para>
+    /// If an explicit <see cref="CookieBuilder.Name"/> is not provided, the system will automatically generate a
+    /// unique name that begins with <c>".AspNetCore.Correlation."</c>.
+    /// </para>
+    /// <para>
+    /// <see cref="CookieBuilder.SameSite"/> defaults to <see cref="SameSiteMode.None"/>.
+    /// <see cref="CookieBuilder.HttpOnly"/> defaults to <c>true</c>.
+    /// <see cref="CookieBuilder.IsEssential"/> defaults to <c>true</c>.
+    /// <see cref="CookieBuilder.SecurePolicy"/> defaults to <see cref="CookieSecurePolicy.SameAsRequest"/>.
```

Comment:
Docs doesn't care about the line breaks. These will get squashed together into one paragraph.

Perhaps use a list? https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/recommended-tags#list

You can preview what XML docs look like by hovering mouse over API or using it from test to see IntelliSense

Reply:
Applies to other lists in this PR.

Reply:
I agree, here is how it looks right now on the one I took for reference.
https://learn.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.antiforgery.antiforgeryoptions.cookie?view=aspnetcore-7.0#remarks